index.bs

```diff
@@ -238,6 +238,22 @@ request could be rejected or even two Picture-in-Picture windows could be
 created. Regardless, the User Agent will have to fire the appropriate events
 in order to notify the website of the Picture-in-Picture status changes.
 
+# Arbitrary Content Specific Concepts # {#arbitrary-concepts}
+
+The Picture-in-Picture window MAY support <dfn>arbitrary content</dfn> which will
+display a {{Document}} in the Picture-in-Picture window. The <a>arbitrary content</a>
+MAY be <dfn>interactive arbitrary content</dfn> which will allow user interaction.
```

Comment:
Shall we define now what is user interaction (keys, gestures, click allowed)?

Reply:
I have defined user interaction briefly, although I have added a TODO since I would like to flesh it out more when I add the security considerations for arbitrary content.